playground/app.vue

```
@@ -1,91 +1,65 @@
-  // a server response via parseApiErrors() and write the result with
-  // setFieldErrors(...) to layer server-side errors on top.
-  const onSubmit = handleSignupSubmit(async (values) => {
-    // simulate server returning a 422
-    const apiResult = parseApiErrors(
-      { error: { details: { email: ['Already taken'] } } },
-      { formKey: signupForm.key }
-    )
-    if (apiResult.ok) setFieldErrors(apiResult.errors)
-    // eslint-disable-next-line no-console
-    console.log('client-validated values:', values)
+  const displayError = computed(() => {
```

ESLint

[error] 'computed' is not defined.

(no-undef)